Drivers/Src/Lights.c

```diff
@@ -9,16 +9,60 @@
 */
 
 
-
 #include "Lights.h"
+#include "BSP_SPI.h"
+#include "stm32f4xx.h"
+#include "Tasks.h"
+
+//static State lightStates[10] = {OFF};
+static uint8_t lightStatesBitmap;
+static uint8_t lightToggleBitmap;
```

Comment:
Leave a comment indicating that this is the enable/disable state of toggleable lights. Only certain bits will ever be high and it corresponds to the toggleable lights.